docs/testing.md

```diff
@@ -40,6 +40,10 @@ To run just a subset of them, specify a particular file or directory:
     py.test tests/unit
     py.test tests/unit/api.py
 
+To run just a pep8 test:
+
+    py.test --pep8 -m pep8
```

Comment:
Given that `.venv` is in our `.gitignore`, let's put the `*` version in the docs.